.github/workflows/chatops.yml

```
@@ -345,8 +345,8 @@ jobs:
           git checkout ${HEAD_BRANCH}
 
           make gotests/install
-          ERR_LOG=$(make gotests/gen 2>&1 >/dev/null)
-          echo "ERR_LOG=$ERR_LOG" >> $GITHUB_OUTPUT
+          make gotests/gen 2>error.log
+          echo "ERR_LOG=$(cat error.log)" >> $GITHUB_OUTPUT
```

I think it looks no problem, but it might be simpler like below suggestion because of no using subshell and temporary file.
```suggestion
          echo -n "ERR_LOG=" 2>> $GITHUB_OUTPUT
          make gotests/gen 2>> $GITHUB_OUTPUT
```

---

Maybe we don't need to pipe error from echo.
```suggestion
                   echo -n "ERR_LOG=" >> $GITHUB_OUTPUT
	          make gotests/gen 2>> $GITHUB_OUTPUT
```